eos-paygd/main.c

```diff
@@ -154,7 +207,10 @@ main (int   argc,
        * initramfs difficult. */
       timeout_id = g_timeout_add_seconds (10 * 60, sync_and_poweroff, NULL);
       while (access ("/etc/initrd-release", F_OK) >= 0)
-        g_usleep (G_USEC_PER_SEC / 5);
+        {
+          g_main_context_iteration (NULL, TRUE); /* keep pinging watchdog */
+          g_usleep (G_USEC_PER_SEC / 5);
```

Comment:
nitpick: I think it'd be clearer if this were expressed like `200 * USEC_PER_MSEC`

Reply:
Honestly to me that seems more esoteric